src/Auth/Actions/userActions.js

```diff
@@ -84,6 +85,34 @@ export const LoginUser = (credentials, history, setFieldError, setSubmitting) =>
     }
 }
 
+
+export const ChangeUserPassword = (credentials, history) => {
+    // Make checks and get some 
```

Comment:
add your api call in its own function in a seperate file, then you can pass the url and method to the function. call me if you don't understand

Reply:
Ok sir